### ISSUE TITLE
Catkin_make error

### ISSUE THREAD

**Issue by zjtgit** [KIT-MRT/pointcloud_surface#1]
`CMake Error at /home/***/catkin_ws/devel/share/uniform_bspline/cmake/uniform_bsplineAutoDepsConfig.cmake:176 (message):
  Package uniform_bspline_eigen was not found.  If it is a catkin package:
  Make sure it is present.  If it is an external package: Make sure it is
  listed in mrt_cmake_modules/yaml/cmake.yaml!
Call Stack (most recent call first):
  mrt_cmake_modules/cmake/Modules/FindAutoDeps.cmake:253 (_find_dep)
  pointcloud_surface/CMakeLists.txt:21 (find_package)

-- Configuring incomplete, errors occurred!
`
I am tring to test the argithm with kitti data, but it seems I can't build the project localy.  The  lib "uniform_bspin_eigen" this project needed, I can't find it here.

**Comment by wsascha** [comment]
Yes, that's right, I couldn't find it either. Maybe @FranklynJey can add it here as I can't do that anymore, unfortunately.

**Comment by wienans** [comment]
Hey the same problem occurs for util_ceres util_eigen util_yaml

**Comment by keroe** [comment]
It takes a really long time for us to open source our internal packages. But we try our best to make the packages available in the next month.

**Comment by aaronxavier** [comment]
@keroe Was this package made open source? I am currently trying to set up pointcloud_surface but this is a roadblock.

**Comment by kgreenek** [comment]
Friendly bump. If the library `uniform_bspline_eigen` were available, that would make this package a lot more useful :) 

As it is, using this library isn't possible without reimplementing large chunks of it.

**Comment by wsascha** [comment]
@keroe @FranklynJey it’d be super awesome if you could publish that lib as it’s also very useful for other tasks. I guess @JohannesBeck wouldn’t object.
